frameworks/CSharp/beetlex/PlatformBenchmarks/DBRaw.cs

```diff
@@ -15,17 +19,14 @@ public class RawDb
 
         private readonly DbProviderFactory _dbProviderFactory;
 
-        private readonly string _connectionString;
+        public static string _connectionString = "Server=192.168.2.19;Database=hello_world;User Id=benchmarkdbuser;Password=benchmarkdbpass;Maximum Pool Size=256;NoResetOnClose=true;Enlist=false;Max Auto Prepare=3";
```

Comment:
Im assuming this connection string is for local tests and is always being overridden in the benchmarks to use `Server=tfb-database`?

Reply:
Use `Server=tfb-database` setting in HttpServer.cs